README.md

```diff
@@ -59,36 +59,65 @@ After configuration is complete, running the link sharing is as simple as:
 $ linksharing run
 ```
 
-## Custom URL configuration and static site hosting*
+## Standard Linksharing with Uplink
+Anything shared with `--url` will be readonly and available publicly (no secret key needed).
```

Comment:
unless you say --readonly=false, right?

Reply:
We only allow url and dns creation `if len(shareCfg.AllowedPathPrefix) == 1 && !permission.AllowUpload && !permission.AllowDelete`

Reply:
And unless I'm missing something, we don't render the page on the linksharing side unless public and readonly as well (or at least only GET requests are permitted)

Reply:
i think the access key doesn't have to be readonly for the linksharing side to work, but it does have to be registered as public.

but okay your other comments answer my question!